### ISSUE TITLE
Package installation on Windows

### ISSUE THREAD

**Issue by snaiperis** [FinYang/tsdl#2]
Hello,

thanks for a nice package containing multiple time series datasets. Suggested installation failed on Windows system. I use tar.exe, so it is available on my system and c:\bin\ is included into PATH. Console output below.

Best regards

> devtools::install_github("FinYang/tsdl")
Downloading GitHub repo FinYang/tsdl@master
c:\bin\tar.exe: Cannot use compressed or remote archives
c:\bin\tar.exe: Error is not recoverable: exiting now
External tar failed with `--force-local`, trying without
c:\bin\tar.exe: Cannot use compressed or remote archives
c:\bin\tar.exe: Error is not recoverable: exiting now
External tar failed with `--force-local`, trying without
Error: Does not appear to be an R package (no DESCRIPTION)
In addition: Warning messages:
1: In utils::untar(tarfile, ...) :
  ‘tar.exe -zxf "C:\Users\user\AppData\Local\Temp\Rtmps7i1PW\file252430421301.tar.gz" -C "C:/Users/user/AppData/Local/Temp/Rtmps7i1PW/remotes25245c5f13c4"’ returned error code 2
2: In system(cmd, intern = TRUE) :
  running command 'tar.exe -ztf "C:\Users\user\AppData\Local\Temp\Rtmps7i1PW\file252430421301.tar.gz"' had status 2

**Comment by robjhyndman** [comment]
This looks like a tar issue, not a problem with the package. See https://superuser.com/questions/124674/mingw-tar-compression-problem for some discussion.

**Comment by snaiperis** [comment]
Thank You, for reply. I managed to solve the problem by manually
downloading tsdl.rda and meta_tsdl.rda files and loading into R.

After your response I rechecked installation with local tar executable
removed. I found that it solves tsdl installation problem. Unexpected
result, but R uses incompatible local tar executable instead of it's own
utils::untar() version, if found. The only suggestion is include package in
CRAN for installation convenience.


Best regards
